src/io.rs

```diff
@@ -303,9 +303,6 @@ pub trait Write {
     /// until completion or return an error.
     fn write_bytes(&mut self, buf: &[u8]) -> Result<(), Error>;
 
-    /// Returns the number of bytes still available to read.
```

Comment:
You can tell I didn't even think about it when I added this because this merely says "bytes available to read"!